distro/travis/before_install.sh

```diff
@@ -0,0 +1,7 @@
+if [ "$TRAVIS_OS_NAME" = "osx" ]; then
```

Comment:
You might want to `set -e` before this so that the script will fail if any line fails